backend/workers/github/src/issues/create.ts

```diff
@@ -0,0 +1,60 @@
+import {z} from "zod";
+import {Octokit, RequestError} from "octokit";
+import {credentials} from "@grpc/grpc-js";
+import parseArguments, {GithubAuthSchema} from "../util/params";
+import {AreaBackServiceClient} from "../proto/area_back";
+import "../proto/google/protobuf/struct";
+import {GithubErrorData} from "../util/types";
+import {onError, onReaction} from "../util/grpc";
```

Comment:
doesn't look linted to me but doesn't really matter

Reply:
You're right! The eslint config didn't trigger prettier